src/pages/index.js

```diff
@@ -308,16 +308,34 @@ function reducer(state, action) {
         buildConfigConfig[action.selectedTab].featureConfig.features
       );
 
+      let shouldSetNoLibrary = state.selectedFeatures['No library'];
+
       const filteredFeatures = _.mapValues(
         state.selectedFeatures,
-        (selected, feature) =>
-          _.includes(newAllPossibleFeatures, feature) && selected
-      );
+        (selected, feature) => {
+          if(
```

Comment:
looks like you didn't run prettier

Reply:
I think this if statement belongs outside the `mapValues` method. You should not need to explicitly set Svelte to false for `filteredFeatures` because Svelte is not part of `allPossibleFeatures` when selecting Parcel.